LDAR_Sim/src/initialization/args.py

```diff
@@ -20,10 +20,35 @@
 # ------------------------------------------------------------------------------
 
 import os
+from pathlib import Path
 from argparse import ArgumentParser, RawTextHelpFormatter
 
 
-def files_from_args():
+def get_abs_path(path, ref_folder=None):
+    """ Get the absolute path of a file from the reference folder
+        If none is specified, use the current working directory
+
+    Args:
+        path (string): relative (or absolute) path to a file
+        ref_folder (Path, optional): reference folder. Defaults to None.
+    """
+    def sel_parent(p_folder):
+        return p_folder.parent
+    if ref_folder is None:
+        ref_folder = Path(os.getcwd())
+    n_pars = len(path) - len(path.lstrip('.'))
+    if n_pars > 0:
+        n_pars -= 1
+        mod_rel_path = path[n_pars:]
+        out_dir = ref_folder
+        for par in range(n_pars):
+            out_dir = sel_parent(out_dir)
+        return out_dir / mod_rel_path
```

Comment:
Yeah, I was having some issues with the resolve, and os.path.abspath with the pathlib library. Which is strange because most people are moving towards pathlib. Ill turn this into an issue, I agree that would be much simpler.

Reply:
#149